.travis.yml

```diff
@@ -23,13 +27,33 @@ jobs:
       script:
         - npx semantic-release
 
+    - stage: e2e
+      language: node_js
+      services:
+        - docker
+      before_install:
+        # Install a newer version of docker-compose than the one provided by Travis
+        - sudo rm /usr/local/bin/docker-compose
+        - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+        - chmod +x docker-compose
+        - sudo mv docker-compose /usr/local/bin
+        - sudo apt-get install libgconf-2-4 # cypress needs this
+        - git clone https://github.com/egendata/e2e.git ../e2e
+      install:
+        - npm ci
+        - cd ../e2e && npm ci
```

Comment:
```suggestion
        npm ci --prefix ../e2e
```

Reply:
We should not be changing directories like this. The change persists after this line